src/lib/djerba/plugins/case_overview/plugin.py

```diff
@@ -38,19 +39,6 @@ class main(plugin_base):
     TUMOUR_DEPTH = 80
     NORMAL_DEPTH = 40
```

Comment:
Normal_depth is 30X (not 40X) not that I see this variable being used anywhere anyways?

Reply:
Can fix this later. I've made a ticket: https://jira.oicr.on.ca/browse/GCGI-1217